python/ray/serve/_private/replica.py

```diff
@@ -815,83 +830,79 @@ async def call_user_method(
         Raises any exception raised by the user code so it can be propagated as a
         `RayTaskError`.
         """
-        async with self._wrap_user_method_call(request_metadata):
```

Comment:
This method is unchanged aside from removing this context manager and un-indenting it. Same with `call_user_method_generator`.